Assets/Scripts/Player/Input Parser.cs

```diff
@@ -0,0 +1,67 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+using FrameWork;
+
+namespace Player.Input_Parser
```

Comment:
```suggestion
namespace Player.Input
```
De namespace en class zouden niet de zelfde naam moeten hebben.